app/install.php

```diff
@@ -362,108 +362,39 @@ function printStep0() {
 <?php
 }
 
+function printStep1Template($key, $value, $messageParams = []) {
+	$message = _t("install.check.{$key}.{$value}", ...$messageParams);
```

Comment:
Could we generalise the translation to have only one string instead of many?

Reply:
We could but I am not sure it will work. I have to think about it.

Reply:
While it would be nice to remove the many translation strings we have for that, it can also wait for later

Reply:
Actually, I think the whole install process needs a bit of love. I'll look into it when I have time and feel like it.

Reply:
Should we then merge this PR as is while waiting for future improvements?

Reply:
I think it's better if you do not want to see this one collect dust